steps/common.py

```diff
@@ -227,3 +227,15 @@ def step(context, cmd, host):
     assert context.remote_cmd(module,
                               host,
                               module_args=args)
+
+@when('checkout the git repo "{repo}" on "{host}"')
+@given('checkout the git repo "{repo}" on "{host}"')
+def step(context, repo, host):
+    "Checkout a git repo on remote host"
+    r = context.remote_cmd('git',
+                            host,
+                            module_args='repo=%s dest=/root/tools' % repo)
+    if r:
+        assert r[0]['changed'] is True, "The checkout of the git repo " + repo + " didn't complete successfully"
+    else:
+        assert False, "The checkout of the git repo " + repo + " didn't complete successfully"
```

Comment:
I think you can just get rid of the `if-then` here and directly do (right after the `remote_cmd()` call):

```python
assert r, "The checkout of the git repo " + repo + " didn't complete successfully"
```